docroot/sites/registrar.uiowa.edu/modules/registrar_core/src/Controller/AcademicCalendarController.php

```diff
@@ -294,13 +334,31 @@ private function processDate($date, $session, $session_index, $session_legacy_id
 
     // Determine the date to display.
     $start_timestamp = strtotime($date->beginDate);
-    $start = date('D, M j, Y', $start_timestamp);
-    $month = date('M', $start_timestamp);
-    $day = date('j', $start_timestamp);
-    if ($date->endDate != $date->beginDate) {
+
+    if ($alt_start_day) {
+      $event->start = $alt_start_day->format('Y-m-d\TH:i:s.000+0000');
+      $alt_start_timestamp = strtotime($event->start);
+      $start = date('D, M j, Y', $alt_start_timestamp);
+      $month = date('M', $alt_start_timestamp);
+      $day = date('j', $alt_start_timestamp);
+    }
+    else {
+      $start = date('D, M j, Y', $start_timestamp);
+      $month = date('M', $start_timestamp);
+      $day = date('j', $start_timestamp);
```

Comment:
some redundancy that could be moved outside the if statement if $alt_start_timestamp and $start_timestamp are assigned to a variable based on the if.

Reply:
Ah, totally. If we come back to this, I will clean it up. I think my coding got dusty while I was working, so I appreciate the cleanup assist.

Reply:
Fixeroni'd!